fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/TopNType.java

```diff
@@ -0,0 +1,29 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.optimizer.operator;
+
+import com.starrocks.thrift.TTopNType;
+
+public enum TopNType {
```

Comment:
The enum of BE is “RankType”，Why not keep same?

Reply:
It's a mistake, I'll change RankType to TopNType